tourism-management-system/Program.cs

```diff
@@ -2,9 +2,64 @@
 {
     internal class Program
     {
-        static void Main(string[] args)
+        static bool CheckCredentials(string position, string password)
         {
-            Console.WriteLine("Hello, World!");
+            switch (position.ToLower())
+            {
+                case "owner":
+                    return password == "ownerpassword";
+                case "receptionist":
+                    return password == "receptionistpassword";
+                case "driver":
+                    return password == "driverpassword";
+                case "photographer":
+                    return password == "photographerpassword";
+                default:
+                    return false;
+            }
+        }
+            static void Main()
+        {
+            Console.WriteLine("Select your position:");
+            Console.WriteLine("1. Owner");
+            Console.WriteLine("2. Receptionist");
+            Console.WriteLine("3. Driver");
+            Console.WriteLine("4. Photographer");
+
+            int choice = Convert.ToInt32(Console.ReadLine());
+
+
+            string position = Position(choice);
+
+            Console.WriteLine("Enter your password:");
```

Comment:
password should be written as a while loop instead of a if loop so the program does not close after the password is written wrong

Reply:
I incorporated the changes suggested by Daniel